superbit_lensing/galsim/config_files/pipe_test.yaml

```diff
@@ -42,10 +42,10 @@ omega_lam:          0.7         # Omega lambda for the background cosmology.
 # Run parameters
 # --------------
 # Define some paths and filenames
-cosmosdir:          '/Users/sweveret/miniconda3/envs/sbmcal/share/galsim/COSMOS_25.2_training_sample/' # Path to COSMOS data directory
-datadir:          '/Users/sweveret/repos/superbit-metacal/superbit_lensing/galsim/data' # Path to repo/galsim data directory
+cosmosdir:          '/Users/jemcclea/Research/SuperBIT/superbit-metacal-v1.0/GalSim/data/' # Path to COSMOS data directory
```

Comment:
I don't know the right way to handle this - we'll just keep overwriting each other's file paths when we merge PR's

Reply:
Sorry, I realize it's a running issue. Let's just revert to the version you have on `master`